Script.js

```diff
@@ -1,18 +1,218 @@
+//  Add click event listener to menu btn to toggle menu
 const expandedNav = document.querySelector('.menu-btn');
 const navLinks = document.querySelector('#desktop_nav');
+const body = document.querySelector('body');
+const workSection = document.querySelector('#Works');
 
-expandedNav.addEventListener('click', () => {
+function mobileMenuClasses() {
   expandedNav.classList.toggle('active');
   expandedNav.classList.toggle('active-menu-btn');
   navLinks.classList.toggle('hide');
   navLinks.classList.toggle('mobile-menu');
+  body.classList.toggle('noscroll');
+}
+
+expandedNav.addEventListener('click', () => {
+  mobileMenuClasses();
 });
 
 navLinks.addEventListener('click', (event) => {
   if (event.target.className === 'hover_effect' && navLinks.classList.contains('mobile-menu')) {
-    expandedNav.classList.toggle('active');
-    expandedNav.classList.toggle('active-menu-btn');
-    navLinks.classList.toggle('hide');
-    navLinks.classList.toggle('mobile-menu');
+    mobileMenuClasses();
+  }
+});
+
+//  Variable to store project objects and properties
+const workProperties = {
+  project1: {
+    featuredImage: 'images/Snapshoot Portfolio.png',
+    name: 'Tonic',
+    client: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    summary: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript'],
+    liveVersionLink: 'Link to live',
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job working on this milestone! 💯 However, in order to improve it, I would suggest you use a simpler data structure. At the moment you have something like this: `{ key: { key: val, ... }, ... }`. This data structure is fine, but it's introducing extra work. If you look closely at the functions that have access to this information, they always convert this to an array so they can iterate through it and manipulate this data. 🤔  Wouldn't it be better to define this as an array of objects and avoid all the extra work? e.g. `[ { key: val, ... } ]` remember to keep things simple. In this case, all of the objects are projects, so they share the same properties! 🤓 🛠️

Reply:
Thank you for the suggestion @anagudelogu. 🤝 
That's a good tip. Accessing the key value pairs would be easier 💯 🙏